### DISCUSSION TITLE
Question: how to track tweaks to BrowserFS

### DISCUSSION THREAD

**Discussion by kumavis** [DustinBrett/daedalOS#300] [General]
since only the built file is commited to the repo, its hard track the changes

**Comment by DustinBrett** [comment]
Yes it is true it's harder to track. I had forked the repo and was working on some branches but there have been a lot of changes recently by other devs on that repo and rather than keep upstream I just had my own local copy. I can attach a diff here for reference. Long term I may update my forked repo rather than include those sources in this repo. But as it's changing I would rather update it before adding to the fork.

[bfs changes.patch](https://github.com/DustinBrett/daedalOS/files/11400935/bfs.changes.patch)

**Comment by kumavis** [comment]
@DustinBrett could you please publish your fork?

**Comment by DustinBrett** [comment]
My fork is a bit fractured and I need to fix it and update what has already been done. It's possible I won't need to fork if BFS is improved, but it's changed a lot and I don't want to focus my attention on BFS atm.

**Comment by james-pre** [comment]
Hey @kumavis, I'm the maintainer of BrowserFS. I'd like to let you know that BrowserFS has been deprecated. You can read more about that on the BrowserFS readme (https://github.com/jvilk/BrowserFS). [ZenFS](https://github.com/zen-fs/core) is an updated and active fork that I created.

What tweaks are you making? Perhaps ZenFS has, or can have, the features you want or at least provide an interface you can use to add that functionality yourself.
